pkg/build/controller/build/podcreationstrategy.go

```diff
@@ -0,0 +1,46 @@
+package build
```

Comment:
AFAICS this code has been copied from pkg/build/controller/factory/factory.go and the original needs to be deleted?